File: app/src/main/java/tech/inscripts/ins_armman/mMitra/utility/IMvpView.java
package tech.inscripts.ins_armman.mMitra.utility;

import android.content.Context;

/**
 * Created by Amol on 11/10/17.
 */

public interface IMvpView {

    Context getContext();

    interface OnFilterResultListener {
        void onZeroResult(String emptyMsg);

        void onMoreThanZeroResult(int resultSize);
    }

}

File: app/src/main/java/tech/inscripts/ins_armman/mMitra/utility/IMvpView.kt
package tech.inscripts.ins_armman.mMitra.utility

import android.content.Context

interface IMvpView {
    fun getContext() : Context
    interface OnFilterResultListener{
        fun onZeroResult(emptyMsg: String)

        fun onMoreThanZeroResult(resultSize: Int)
    }
}